config/jms/activemq-jms-config.yaml

```diff
@@ -65,7 +65,7 @@ servers:
                 - "[[.KieName]]-amq"
               from:
                 kind: ImageStreamTag
-                name: "amq-broker-72-openshift:1.3"
+                name: "amq-broker-73-openshift:7.3"
```

Comment:
Same here... @tchughesiv ?

Reply:
might be a good idea yea

Reply:
any suggestion about where should I add the constant? I mean, create a new struct or use a existing one?

Reply:
probably best to add a new (couple?) of strings here:
https://github.com/kiegroup/kie-cloud-operator/blob/80b536c63fa33d39d626e590961a12cc94cb91b9/pkg/apis/app/v1/kieapp_types.go#L335

something like `brokerImage` & `brokerImageTag`?
You'll have to populate this Constants section, likely in this function or another in defaults.go...
https://github.com/kiegroup/kie-cloud-operator/blob/80b536c63fa33d39d626e590961a12cc94cb91b9/pkg/controller/kieapp/defaults/defaults.go#L195

maybe create an actual constant you can use to populate w/ @
https://github.com/kiegroup/kie-cloud-operator/blob/80b536c63fa33d39d626e590961a12cc94cb91b9/pkg/controller/kieapp/constants/constants.go#L8